app/src/ui/repositories-list/group-repositories.ts

```diff
@@ -28,18 +45,24 @@ export function groupRepositories(
   localRepositoryStateLookup: ReadonlyMap<number, ILocalRepositoryState>
 ): ReadonlyArray<IFilterListGroup<IRepositoryListItem>> {
   const grouped = new Map<RepositoryGroupIdentifier, Repositoryish[]>()
+  const gitHubOwners: Array<string> = []
```

Comment:
Could this be a `Set<string>`? We'd get "avoid duplicate entries" for free, which is helpful later on.